docs/markdown/Using Pants/concepts/options.md

```diff
@@ -81,23 +81,33 @@ Note that any dashes in the option flag name are converted to underscores: `--mu
 
 ### Config file interpolation
 
-Environment variables can be interpolated by using the syntax `%(env.ENV_VAR)s`, e.g.:
+A string value in a config file can contain placeholders of the form `%(key)s`, which will be replaced with a corresponding value. The `key` can be one of:
```

Comment:
This correctly documents what were already doing in Python (and are now doing in Rust). The documentation was previously inaccurate.